src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -107,7 +107,7 @@ static DataTestUtility()
             }
 
             AKVOriginalUrl = c.AzureKeyVaultURL;
-            if (!string.IsNullOrEmpty(AKVOriginalUrl) && Uri.TryCreate(AKVOriginalUrl, UriKind.Absolute, out Uri AKVBaseUri))
```

Comment:
This was creating a new local variable and not updating the static field in DataTestUtility. When the static `AKVBaseUri` was used later to instantiate a `KeyClient`, it was null.